wger/core/views/user.py

```diff
@@ -508,11 +508,14 @@ def get_queryset(self):
         Return a list with the users, not really a queryset.
         '''
         out = {'admins': [],
-               'members': []}
+               'active_members': [],
+               'deactivated_members': []}
+        specific_user = User.objects.select_related('usercache', 'userprofile__gym')
 
-        for u in User.objects.select_related('usercache', 'userprofile__gym').all():
-            out['members'].append({'obj': u,
-                                   'last_log': u.usercache.last_activity})
+        for u in specific_user.filter(is_active=True):
+            out['active_members'].append({'obj': u, 'last_log': u.usercache.last_activity})
+        for u in specific_user.filter(is_active=False):
+            out['deactivated_members'].append({'obj': u, 'last_log': u.usercache.last_activity})
```

Comment:
@Rhytah do we have to make two loops? How about an if statement within one loop to append to the appropriate key in the `out` dictionary? Let me know why you decided to go with two loops instead.

Reply:
@AGMETEOR at the time, it seemed to be the most straight forward approach to filtering the users into separate lists. Let me try using your recommendation and get back to you.

Reply:
@AGMETEOR I have refactored this to have one for loop.